File: java/lista_3.java
import java.util.HashMap;
import java.util.Map;

public class lista_3 {
    public static void numberOfOccurrences(String input) {
        String[] words = input.toLowerCase().split("\\s+");
        Map<String, Integer> wordCountMap = new HashMap<>();

        for (String word : words) {
            wordCountMap.put(word, wordCountMap.getOrDefault(word, 0) + 1);
        }

        for (Map.Entry<String, Integer> entry : wordCountMap.entrySet()) {
            System.out.println("Word: '" + entry.getKey() + "' occurs " + entry.getValue() + " times.");
        }
    }

    public static void main(String[] args) {
        String input = "To jest przykladowe zdanie. To zdanie jest tylko przykladowe.";
        numberOfOccurrences(input.toLowerCase());
    }
}

File: kotlin/lista_3.kt
fun numberOfOccurrences(input: String) {
    val words = input.lowercase().split("\\s+".toRegex())
    val wordCountMap = mutableMapOf<String, Int>()

    for (word in words) {
        wordCountMap[word] = wordCountMap.getOrDefault(word, 0) + 1
    }

    for ((word, count) in wordCountMap) {
        println("Word: '$word' occurs $count times.")
    }
}

fun main() {
    val input: String = "To jest przykladowe zdanie. To zdanie jest tylko przykladowe."
    numberOfOccurrences(input.lowercase())
}

